src/metabase/task/send_pulses.clj

```diff
@@ -0,0 +1,120 @@
+(ns metabase.task.send-pulses
+  "Tasks related to running `Pulses`."
+  (:require [clojure.tools.logging :as log]
+            [clj-time.core :as time]
+            [korma.core :refer :all]
+            [metabase.db :refer :all]
+            [metabase.driver :as driver]
+            ;[metabase.email.messages :refer [send-email-report]]
+            ;(metabase.models [pulse :refer [EmailReport
+            ;                                      days-of-week
+            ;                                      execution-details-fields
+            ;                                      mode->id
+            ;                                      time-of-day->realhour]]
+            ;                 [emailreport-executions :refer [EmailReportExecutions]]
+            ;                 [pulse-channel-recipient :refer [EmailReportRecipients]]
+            ;                 [hydrate :refer :all])
+            [metabase.task :as task]
+            [metabase.util :as u]))
+
+
+;(declare execute-scheduled-reports
+;         execute-if-scheduled
+;         execute-and-send
+;         execute
+;         report-fail
+;         report-complete)
+;
+;(defn execute-reports-hourly-job
```

Comment:
yeah, this is just placeholder right now.  i commented out this whole file briefly so that i wouldn't get compile warnings and i'm going to come back to it shortly and actually fix it.